### PR TITLE
Top menu: Add link to view current user's profile

### LINKED ISSUE #853: Add navigation bar link to "update profile"
Navigation menu currently doesn't link to profile update page - it should be under "Hi, name".

### PR DESCRIPTION
**Pull request by pbanaszkiewicz**
This fixes #853.

### CODE REVIEW

**Comment by pbanaszkiewicz** [comment]
Future-proofing when #856 gets available for no-admin users.

File: workshops/templates/navigation_fixed.html [hunk 1]
```diff
@@ -95,6 +95,11 @@
             <li class="dropdown">
               <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Hi, {{ user.get_short_name }} <span class="caret"></span></a>
               <ul class="dropdown-menu" role="menu">
+                {% if user.is_superuser %}
+                <li><a href="{% url 'person_details' user.id %}">Your profile</a></li>
```
Comment (gvwilson) [inline workshops/templates/navigation_fixed.html]: Is "Your profile" the correct title, when it's (probably) someone else's?

Comment (pbanaszkiewicz) [inline workshops/templates/navigation_fixed.html]: How?

File: workshops/templates/navigation_fixed.html [hunk 2]
```diff
@@ -95,6 +95,11 @@
             <li class="dropdown">
               <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Hi, {{ user.get_short_name }} <span class="caret"></span></a>
               <ul class="dropdown-menu" role="menu">
+                {% if user.is_superuser %}
```
Comment (chrismedrela) [inline workshops/templates/navigation_fixed.html]: Why not `user.is_admin`?

**Comment by chrismedrela** [comment]
:+1 except for this one issue that I raised.

**Comment by pbanaszkiewicz** [comment]
@chrismedrela I went with your suggestion + I slightly optimized `person.is_admin` for superusers.

**Pull request merged**